src/ia-book-search-results.js

```diff
@@ -11,6 +11,9 @@ export class IABookSearchResults extends LitElement {
 
   static get properties() {
     return {
+      bookReader: {
```

Comment:
General question about these menus... do they all communicate directly with the book reader or is there a communication bridge between them? 
ie 
"Menu interaction emits event (search selected)" -> "Controller listens for that event" -> "Tells book reader to do something"
vs
"Menu interaction" -> "Call method on BR"

Reply:
I suggested adding event listeners to BookReader so we could trigger events rather than know the inner workings of BookReader a few weeks ago, and Drini wanted to leave BookReader as is and instead call the methods directly. The method calls could exist outside of this component in a controller within BookNavigator & Petabox instead. I'm doing the same with the actual search submission and loading search results into this component.

Reply:
Yeah, I'm going to move them to happen on event triggering. Lack of sleep and coding too early had me inserting it directly into the component. Thanks for the catch!

Reply:
Updated to remove BookReader prop and method calls. Added a quick console log event listener for the resultSelected event in the demo.